pkg/web/router/router.go

```diff
@@ -125,6 +128,5 @@ func SetAppRoutes(r *echo.Group, appsHandler apps.HTTPHandler) {
 	//     description: Invalid appId supplied
 	//   404:
 	//     description: App not found
-	r.POST("/init", appsHandler.GetApps) // TODO: Implement the func which will be used here and its call
-
```

Comment:
- Here shows are missing updating the swagger doc comments and re-generated the swagger API. You can use the make test,